src/clj/game/cards/resources.clj

```diff
@@ -862,6 +866,32 @@
    "Laguna Velasco District"
    {:events {:runner-click-draw {:msg "draw 1 card" :effect (effect (draw))}}}
 
+   "Lewi Guilherme"
+   (let [ability {:once :per-turn
+                  :delayed-completion true
+                  :effect (req (if (zero? (:credit runner))
```

Comment:
If runner has credits on Net Mercur they could still lose a credit I believe, even if their credit count is 0.

Reply:
That's what I get for trying to be cute.